pages/index.tsx

```diff
@@ -36,38 +36,24 @@ const htmrTransform: HtmrOptions["transform"] = {
 
 interface LastUpdatedAlertProps {
   className?: string;
-  style?: React.CSSProperties;
 }
 
-const LastUpdatedAlert = ({ className, style }: LastUpdatedAlertProps) => (
-  <div
-    className={clsx(
-      "bg-yellow-50 border-l-4 border-yellow-400 p-4 sm:rounded",
-      className,
-    )}
-    style={style}
-  >
-    <div className="flex">
-      <div className="flex-shrink-0">
-        <ClockIcon aria-hidden="true" className="h-5 w-5 text-yellow-400" />
-      </div>
-      <div className="ml-3">
-        <p className="text-sm text-yellow-700">
-          Pembaruan terakhir pada{" "}
-          {new Date(attributes.last_updated_time).toLocaleString("id", {
-            weekday: "long",
-            year: "numeric",
-            month: "long",
-            day: "numeric",
-            hour: "2-digit",
-            minute: "2-digit",
-            timeZone: "Asia/Jakarta",
-            timeZoneName: "short",
-          })}
-        </p>
-      </div>
-    </div>
-  </div>
+const LastUpdatedAlert = ({ className }: LastUpdatedAlertProps) => (
+  <Alert accentBorder className={className} icon={ClockIcon}>
```

Comment:
This line is also needs to be covered by your tests.

Reply:
I have added a test for the `HomePage`, but apparently it's not sufficient to test all the possibilities in the `helpers.ts` file. 😅